src/HttpMultipart/HttpFoundation/MultipartResponse.php

```diff
@@ -5,8 +5,7 @@
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 
-class MultipartResponse extends Response
-{
+class MultipartResponse extends Response {
```

Comment:
In the HttpMultipart directory we have a separate library - https://github.com/dickolsson/http-multipart. We'll probably remove it and import in other way. Let's keep the code standard as it is, for the moment.